Added black,

MeGaGiGaGon opened the pull request:
Also did the flake8 and isort compatibility

MeGaGiGaGon commented:
@A5rocks For the flake8 config, should we have it as it's own file or use https://pypi.org/project/Flake8-pyproject/
Should we also move the isort config to pyproject.toml?

A5rocks commented:
I guess using flake8-pyproject is nicer so we have everything in one place.

And yeah also isort.

MeGaGiGaGon commented:
@A5rocks I think that should be the things, can you make sure?

A5rocks reviewed (approved):
Small nitpick but you can ignore it if you want (let's discuss how to merge this in the channel when you're ready!)

A5rocks commented:
(sorry about the spam, trying to get the webhook to work :( )

MeGaGiGaGon commented:
Proposed merge message:

- Added support for the Black formatter on pre-commit
- Configured isort and flake8 to work with black
- Added flake8-pyproject so the flake8 configs could be moved to pyproject.toml

Still todo:

- Fix/Update module versions so that flake8-pyproject can be pinned
- Fix config settings in tox.ini
